src/streaming-services/netflix/NetflixApi.ts

```diff
@@ -56,7 +55,7 @@ export interface NetflixHistoryResponse {
 
 export type NetflixHistoryItem = NetflixHistoryShowItem | NetflixHistoryMovieItem;
 
-export interface NetflixHistoryShowItem {
+export interface NetflixHistoryShowItem extends HistoryItem {
```

Comment:
Are you sure that this won't open up the interface to basically anything?

Reply:
Not sure. I guess we could use `unknown` instead of `HistoryItem` in the `Api` class, because the type doesn't really matter there.